docs/ReleaseNotes_12.0.md

```diff
@@ -0,0 +1,89 @@
+# Version 12.0.rc1 Beta
+This is the beta for the version 12.0 release. We will be in beta for about a week unless major issues are discovered.
+
+## New Features
+- Created basic layer movement functionality (up, down) by @affirVega in https://github.com/flameshot-org/flameshot/pull/2108
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/layer.png" width=50%> </p>
+
+- Added a new widget to allow the colorwheel to be more easily customized by https://github.com/flameshot-org/flameshot/pull/2202
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/colorwheel.png" width=50%> </p>
+
+
+- Added magnifier for more precise selections by @SilasDo in https://github.com/flameshot-org/flameshot/pull/2219
+  - The new magnifier can be enabled in ```Configuration > General > Show Magnifier``` 
+  - There is an option to make the magnifier a square or circle
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/magnfier.png" width=50%> </p>
+
+- Incremental markers can now have a point if you drag when placing them. @vozdeckyl in https://github.com/flameshot-org/flameshot/pull/2638
+<p align=center><img src="https://github.com/flameshot-org/flameshot/blob/master/docs/images/number_pointer.png" width=50%> </p>
+
+- Added the ability to cache the last region by @borgmanJeremy in https://github.com/flameshot-org/flameshot/pull/2615
+  - The launcher tool will automatically populate the coordinates for the last selection region
```

Comment:
I suggest adding a screenshot for this.

For example:

![image](https://user-images.githubusercontent.com/390889/173530450-b6b79f8b-81a0-404a-881f-98bc1e44aa4f.png)